github.go

```diff
@@ -83,10 +83,16 @@ func slurpFile(url string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	defer resp.Body.Close()
+	defer func() {
+		err = resp.Body.Close()
+	}()
+	if err != nil {
+		return "", err
+	}
```

Comment:
Just ignore the error. If you `defer` here the if condition afterwards never evaluates to something different than false.

```suggestion
	defer func() {
		_ = resp.Body.Close()
	}()
```

Reply:
decided to use the second suggestion as appears to be cleaner to me